i18n/en.json

```diff
@@ -536,6 +536,38 @@
     "translation": "echo"
   },
   {
+    "id": "api.command_channel_header.desc",
+    "translation": "Edit the channel header."
```

Comment:
Can you help remove the period at the end? I.e. `Edit the channel header`

This is consistent with the other autocomplete help descriptions.